lib/sel/sel.ts

```diff
@@ -209,8 +209,8 @@ export const sel: Sel = new Proxy(
             {
               get: (_, pinOrSubComponentName: string) => {
                 const pinResult = `.${prop1} > .${pinOrSubComponentName}`
-
-                if (prop1.startsWith("U")) {
+                const chipPrefixes = ["U", "J"]
```

Comment:
the JumperSel specifies CN is also valid

```suggestion
                const chipPrefixes = ["U", "J", "CN"]
```